docs/config.toml

```diff
@@ -78,11 +78,12 @@ home = [ "HTML", "RSS", "SearchIndex" ]
     { name = "StarRocks", identifier = "_starrocks", weight = 701, url = "https://docs.starrocks.io/en-us/latest/data_source/catalog/iceberg_catalog" },
     { name = "Amazon Athena", identifier = "_athena", weight = 800, url = "https://docs.aws.amazon.com/athena/latest/ug/querying-iceberg.html" },
     { name = "Amazon EMR", identifier = "_emr", weight = 900, url = "https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-use-cluster.html" },
-    { name = "Impala", identifier = "_impala", weight = 1000, url = "https://impala.apache.org/docs/build/html/topics/impala_iceberg.html" },
-    { name = "Doris", identifier = "_doris", weight = 1001, url = "https://doris.apache.org/docs/dev/lakehouse/multi-catalog/iceberg" },
-    { name = "PyIceberg", identifier = "_python", weight = 1500, url = "https://py.iceberg.apache.org/"},
-    { name = "Integrations", weight = 1100 },
-    { name = "API", weight = 1200},
-    { name = "Migration", weight = 1300},
-    { name = "Javadoc", pre = "relative", url = "../../javadoc/latest", weight = 1400} # `url` is populated by the github deploy workflow and is equal to "../../javadoc/<branch name>"
+    { name = "Snowflake", identifier = "_snowflake", weight = 1000, url = "https://docs.snowflake.com/en/user-guide/tables-iceberg" },
+    { name = "Impala", identifier = "_impala", weight = 1100, url = "https://impala.apache.org/docs/build/html/topics/impala_iceberg.html" },
+    { name = "Doris", identifier = "_doris", weight = 1200, url = "https://doris.apache.org/docs/dev/lakehouse/multi-catalog/iceberg" },
+    { name = "PyIceberg", identifier = "_python", weight = 1700, url = "https://py.iceberg.apache.org/"},
```

Comment:
nit: While at it, should we move PyIceberg to the bottom, since it has the highest weight?

Reply:
Yeah, I couldn't tell what the ordering logic was, also why weights were what they were. Thanks for the cleanup.